features/lwipstack/LWIPInterface.cpp

```diff
@@ -457,6 +495,30 @@ nsapi_error_t LWIP::add_l3ip_interface(L3IP &l3ip, bool default_if, OnboardNetwo
 nsapi_error_t LWIP::remove_l3ip_interface(OnboardNetworkStack::Interface **interface_out)
```

Comment:
As `add_l3ip_interface` calls `power_up` for the driver (via `l3ip_if_init`), should similarly `remove_l3ip_interface` call `power_down`?

Reply:
This should be performed by driver

Reply:
on destruction

Reply:
However  it is reasonable to add   power_down in remove_l3ip_interface. 
I will add